package.json

```diff
@@ -73,6 +77,8 @@
     "sharp": "^0.31.0",
     "source-map-loader": "^4.0.1",
     "style-loader": "3.3.2",
+    "ts-jest": "^26.5.0",
+    "typescript": "^4.9.4",
```

Comment:
[clarification] Should `typescript` be removed from the `devDependencies` given its now installed in `dependencies`?